setup

```diff
@@ -0,0 +1,84 @@
+whoisthis="Eric Harrison"  #your name, used for Git config
```

Comment:
current best practices are to leave bash scripts without an extension generally, but you should include a line like this to start the script: ` #!/bin/bash`
```suggestion
#!/bin/bash
whoisthis="Eric Harrison"  #your name, used for Git config
```